frontend/signup.html

```diff
@@ -110,21 +110,53 @@ <h2>Sign Up</h2>
           <button onclick="saveUsername()">Sign Up</button>
           <div class="login">
             Already have an account?
-            <a class="login-link" href="./login.html">Login</a>
+            <a class="login-link" href="./login_mongo.html">Login</a>
```

Comment:
add this line else it doesnt call login page
```suggestion
            <a class="login-link" href="./login_mongo.html">Login</a>
```